python/tests/unit/arcticdb/version_store/test_sort_merge.py

```diff
@@ -612,4 +614,41 @@ def test_update_symbol_list(lmdb_library):
     lib.write(sym_2, df, staged=True)
     lib.sort_and_finalize_staged_data(sym_2, mode=StagedDataFinalizeMethod.APPEND)
     assert lib_tool.count_keys(KeyType.SYMBOL_LIST) == 3
-    assert set(lib.list_symbols()) == set([sym, sym_2])
\ No newline at end of file
+    assert set(lib.list_symbols()) == set([sym, sym_2])
+
+class TestSlicing:
```

Comment:
Can we also have a test where the individual staged segments are smaller than the segment row count, but when they are compacted they are larger?